README.md

```diff
@@ -27,10 +27,15 @@ repositories {
 }
 
 dependencies {
-    'com.basecamp:turbolinks:1.0.2'
+    compile 'com.basecamp:turbolinks:1.0.2'
 }
 ```
 
+Add the INTERNET permission to `AndroidManifest.xml`
```

Comment:
The reason I added this is because no Exception is raised and the log output from TurbolinksSession doesn't even hint as to this being a problem. I spent a good half hour assuming I had messed something up in the quick start guide and had to resort to diffing my app against the demo app to discover the missing perm. If you'd prefer to raise an Exception instead, I probably could submit a PR for that